widgets/htdocs/widgets/95_SpeciesTree.js

```diff
@@ -15,7 +15,7 @@ Ensembl.SpeciesTree.displayTree = function(json, panel) {
   var theme =  Ensembl.SpeciesTree.tnt_theme_tree_simple_species_tree(json);
   theme(tnt.tree(), document.getElementById("species_tree"));
 },
-
+Ensembl._species_tree_json;
```

Comment:
I cant see where this is used again. it looks like we dont need this variable or am i missing something